java-neg-subarray

```diff
@@ -0,0 +1,35 @@
+import java.io.*;
+import java.util.*;
+
+public class Solution {
+
+    public static void main(String[] args) {
+        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
+        
+         Scanner sc = new Scanner(System.in);
```

Comment:
call sc.close() or even better to use try-with-resource solution since Scanner implements AutoClosable